src/tests/ftest/datamover/dm_negative.py

```diff
@@ -279,11 +279,14 @@ def test_dm_negative_space_dcp(self):
         # Start dfuse on pool2/cont2
         self.start_dfuse(self.dfuse_hosts, pool2, cont2)
 
+        # create posix destination path in self.tmp 
+        posix_dst_path = self.new_posix_test_path()
+
```

Comment:
test_dm_negative_space_dcp is tricky:
1. This test is commented out, so it won't run anyway.
2. We DON'T want to use self.tmp or self.workdir here, because if we run out of space, it could cause other issues, since the filesystem is used by other tests.
3. https://daosio.atlassian.net/browse/DAOS-6871 exists to re-enable this test.

I recommend just leaving this test as-is for now, since it will be skipped anyway.

Reply:
@daltonbohning thanks, yeah I wasn't sure about this one because it is skipped.